README.md

```diff
@@ -1,78 +1,90 @@
-* [Tutorials](http://geomesa.github.io)
+* [Quick Start](http://www.geomesa.org/geomesa-quickstart/) on the main [documentation](http://www.geomesa.org/) site.
+* [FAQ](http://www.geomesa.org/faq/)
+* [Tutorials](http://www.geomesa.org/tutorials/)
 * GeoMesa [Users](https://locationtech.org/mhonarc/lists/geomesa-users/) and [Dev](https://locationtech.org/mhonarc/lists/geomesa-dev/) mailing list archives
 * READMEs are provided under most modules: [Tools](geomesa-tools), [Jobs](geomesa-jobs), etc
 
 ## GeoMesa Project Structure
 
-#### geomesa-core
-
-This project contains the implementations of the core indexing structures, Accumulo iterators, and the GeoTools interfaces for exposing the functionality as a ```DataStore``` to both application developers and GeoServer.
-
-##### Scala console via scala-maven-plugin
-
-To test and interact with core functionality, the Scala console can be invoked in a couple of ways.  From the root directory by specifying geomesa-core ```geomesa> mvn -pl geomesa-core scala:console```.  Or from the sub-project's directory ```geomesa-core> mvn scala:console```.  By default, all of the project packages in ```core``` are loaded along with JavaConversions, JavaConverters.
-
-#### geomesa-distributed-runtime
-
-This sub-project assembles a jar with dependencies that must be distributed to Accumulo tablet servers lib/ext
-directory or to an HDFS directory where Accumulo's VFSClassLoader can pick it up.
-
-#### geomesa-plugin
-
-This sub-project creates a plugin which provides WFS and WMS support.  The JAR named geomesa-plugin-<Version>-geoserver-plugin.jar is ready to be deployed in GeoServer by copying it into geoserver/WEB-INF/lib/
-
-#### geomesa-utils
-
-This sub-project stores our GeoHash implementation and other general library functions unrelated to Accumulo. This sub-project contains any helper tools for geomesa.  Some of these tools such as the GeneralShapefileIngest have Map/Reduce components, so the geomesa-utils JAR lives on HDFS.
-
-#### geomesa-compute
-
-This sub-project contains utilities for working with distributed computing environments.  Currently, there are methods for instantiating an Apache Spark Resilient Distributed Dataset from a CQL query against data stored in GeoMesa.  Eventually, this project will contain bindings for traditional map-reduce processing, Scalding, and other environments.
-
-#### geomesa-tools
-
-This sub-project contains a set of command line tools for managing features, ingesting and exporting data, configuring tables, and explaining queries in GeoMesa. Please view the [geomesa-tools README](https://github.com/locationtech/geomesa/tree/master/geomesa-tools) to learn more.
+* [**geomesa-accumulo**](geomesa-accumulo/geomesa-accumulo-datastore): This module contains the implementations of the core Accumulo indexing structures, Accumulo iterators, and the GeoTools interfaces for exposing the functionality as a `DataStore` to both application developers and GeoServer.
```

Comment:
I like the new layout and the links.  Maybe reduce the amount of 'This module contains' repeated language?  @cne1x  do you have any wording opinions?